build/vega-lite-schema.json

```diff
@@ -1952,6 +1964,45 @@
           "description": "The encoded field's type of measurement. This can be either a full type\nname (`\"quantitative\"`, `\"temporal\"`, `\"ordinal\"`,  and `\"nominal\"`)\nor an initial character of the type name (`\"Q\"`, `\"T\"`, `\"O\"`, `\"N\"`).\nThis property is case-insensitive."
         }
       },
+      "required": [
+        "type"
+      ],
+      "type": "object"
+    },
+    "FieldDefBase": {
```

Comment:
This is not really referenced, but the new generator somehow already refers to every extended class.  (This is a known bug, but does not harm besides making the schema a bit unnecessary big.)